roles/keystone/meta/main.yml

```diff
@@ -14,15 +14,17 @@ dependencies:
     alternatives: "{{ keystone.alternatives }}"
     system_dependencies: "{{ keystone.source.system_dependencies }}"
     python_dependencies: "{{ keystone.source.python_dependencies }}"
+    python_post_deps: "{{ keystone.source.python_post_deps }}"
```

Comment:
should we spell `dependencies` out fully to be inline with the other variable names ?